site/tailwind.config.js

```diff
@@ -10,11 +10,15 @@ export default {
   important: true,
   safelist: ['dark'],
   theme: {
+    fontFamily: {
+      sans: ['Inter', 'sans-serif'],
+    },
```

Comment:
Override the default `fontFamily` config.

Reply:
Ok dope